pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -1262,6 +1262,35 @@ func (c *Cluster) startMon(m *monConfig, schedule *MonScheduleInfo) error {
 		logger.Debugf("adding host path volume source to mon deployment %s", d.Name)
 	}
 
+	// Expand mon PVC if storage request for mon has increased in cephcluster crd
```

Comment:
The expansion likely won't be picked up until the next mon pod restart. The way we handled this for OSDs is that we added a label to the osd pod that is the size of the PVC. Then if the size changes, the pod spec will be updated with the new size and the pod will naturally be restarted.

Reply:
Done